rafthttp/peer.go

```diff
@@ -26,7 +26,13 @@ import (
 )
 
 const (
-	DialTimeout      = time.Second
+	DialTimeout = time.Second
+	// ConnRead/WriteTimeout is the i/o timeout set on each connection rafthttp pkg creates.
+	// For long term streaming connections, rafthttp pkg sends application level linkHeartbeat
+	// to keep the connection alive.
+	// For short term pipeline connections, rafthttp MUST kill the connection to avoid it being
+	// put back to http pkg connection pool.
```

Comment:
could we also mention this in newTimeoutTransport, which these consts are used?https://github.com/coreos/etcd/blob/master/pkg/transport/timeout_transport.go#L26
I think they are the root cause that we need to do it.